stingray/utils.py

```diff
@@ -119,13 +121,14 @@ def rebin_data(x, y, dx_new, method='sum'):
 def assign_value_if_none(value, default):
     return default if value is None else value
 
-
 def look_for_array_in_array(array1, array2):
     return next((i for i in array1 if i in array2), None)
 
-
 def is_string(s):  # pragma : no cover
-    """Portable function to answer this question."""
+    """
```

Comment:
The old version of the docstring was more compliant with PEP 257: https://www.python.org/dev/peps/pep-0257/